lib/to-es5.js

```diff
@@ -13,7 +13,7 @@ function ImportExternalHelpersPlugin() {
   };
 }
 
-ImportExternalHelpersPlugin.baseDir = function() { return __dirname; };
+ImportExternalHelpersPlugin.baseDir = function() { return process.cwd(); };
```

Comment:
this isn't correct, we will be scanning and include unexpected things in the cache key. Disable the persistent cache instead, until the root cause is understood.